kotsadm/web/src/components/apps/AppGitops.jsx

```
@@ -105,33 +102,22 @@ class AppGitops extends Component {
           Utilities.logoutUser();
           return;
         }
-        let msg = `Unexpected status code: ${res.status}`;
+        this.props.refetch();
+
         if (res.status === 400) {
-          msg = `Unable to authenticate. Please make sure you have added your deployment key to the git repo.`;
-        }
-        try {
-          const data = await res.json();
-          if (data?.error) {
-            console.log(`Failed to test gitops connection: ${data.error}`);
+          const response = await res.json();
+          if (response?.error) {
+            console.log(response?.error);
```

this error is not surfaced in the UI

---

its pretty technical